### PR TITLE
Adding KubeCon NA 2023 banner

### PR DESCRIPTION
**Pull request by nate-double-u**
Adding KubeCon NA 2023 banner:

Screenshot 2023-10-18 at 11 46 28 AM

scheduled between 2023-10-20 and 2023-11-09

resolves https://github.com/kubernetes/website/issues/43564

/committee steering
@kubernetes/steering-committee

### CODE REVIEW

**Comment by reylejano** [comment]
/lgtm

This needs @kubernetes/steering-committee approval

File: data/announcements/scheduled.yaml
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:181]: _nit:_ Could we possibly move this new announcement block to the end of the file? This way, the announcements in the file would be in some _(chronological)_ order for better readability, even though I know that we have added some older announcements in between out of order to accommodate scheduling preferences for overlapping dates.

Comment (nate-double-u) [reply data/announcements/scheduled.yaml]: Good call, updated.

**Review by Arhell — COMMENTED** [review]
/lgtm

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
/hold

**Review by onlydole — APPROVED** [review]
LGTM! Thank you, @nate-double-u - feel free to remove the hold once you're ready!

**Comment by nate-double-u** [comment]
Thanks everyone!
/remove-hold

**Pull request merged**
